### DISCUSSION TITLE
Taunt cooldown removed while solo. Range limit to taunt cooldown.

### DISCUSSION THREAD

**Discussion by TCUPfromOLDdota2DOTdev** [ValveSoftware/Dota2-Gameplay#12177] [Ideas]
Getting a rune, taunting, waiting for a gank, taunting, waiting for camp stacks, taunting. Dancing, eating, flipping eggs, rolling around. Please enable spamming of taunts when not near anyone. So i can dance and keep myself amused. 

Such a simple joy and pleasure to taunt nonstop while dotaing. I beseech thee dota Gods. 

If its too hard then i understand. I will keep reminising of the days i could dance non stop. <3

**Comment by run3unblockedaz** [comment]
Hi! After reading a few of the postings, I realized that I had never been to this website before, despite the fact that I could have sworn that I had been here before. On the other hand, I can't express how happy I am that I discovered  <a href="https://monkeymartgame.com/main">**monkey mart**</a>. en and make sure to check back!
